treatment-service/models/storage_test.go

```diff
@@ -528,6 +528,9 @@ func TestDumpExperiments(t *testing.T) {
 			1: {NewExperimentIndex(e)},
 		},
 	}
+	// Need to hardcode this to testdata/experiments_dump.json ExperimentId
+	// because we are randomly generating Id in newTestXPExperiment call
+	e.Id = 81
```

Comment:
I see.. Can we instead make the function `newTestXPExperiment` accept a parameter to explicitly pass in the ID we want to use ?

Reply:
@terryyylim, I believe this comment is not addressed yet. Could you check?

Reply:
Hmm actually my bad - I see the function `newTestXPExperiment` is modified.

So why is this still needed?

<img width="947" alt="Screenshot 2023-05-15 at 8 07 55 AM" src="https://github.com/caraml-dev/xp/assets/23465343/192f546f-372f-4b46-b81e-45380a13af2d">

Reply:
Oh, which part would you be referring to? Passing in `81`?

Reply:
Yes

Reply:
Ahh I think I got it now - the 81 is now the parameter. Looks like the comment above that is outdated - it's no longer any "hardcoding", right?

Reply:
Yes correct!